js/ui/tab_cookies.js

```diff
@@ -164,7 +200,7 @@
         break;
       case "delete-domain-cookies":
         var domain = getParentData(element, "data-access");
-        getAllCookies({"domain": domain}, function(cookies)
```

Comment:
Why was domain cookies removal functionality removed ? This shouldn't be necessary.
This is triggered when user hit delete key when focused on a top level list item(Domain item) or when "x" button next to the list item is clicked, see -> https://github.com/Manvel/Privacy-Manager/blob/master/popup.html#L113